modular_skyrat/modules/cryosleep/code/cryopod.dm

```diff
@@ -340,12 +340,29 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		return
 
 	if(target.key && user != target)
-		if(iscyborg(target))
+		var/mob/living/mob_occupant = target //Get mob living
+		if (target.getorgan(/obj/item/organ/brain) ) //Target the Brain
+			if(ai_controller?.ai_status == AI_STATUS_ON) // Is the character empty / AI Controlled
```

Comment:
Check for a lack of client in an OR statement. This also allows for cryoing of `/mob/living`s that aren't animals but are AI controlled.